### PR TITLE
Added as contributor.

### PR DESCRIPTION
**Pull request by jlatherfold**

### CODE REVIEW

File: CONTRIBUTORS.txt [hunk 1]
```diff
@@ -154,6 +154,9 @@ In chronological order:
 * Jon Wayne Parrott <jonwayne@google.com>
   * App Engine environment tests.
 
+* James Atherfold <jlatherfold@hotmail.com>
+  * Bugfixes.
```
Comment (shazow) [inline CONTRIBUTORS.txt]: Can you be a biiit more descriptive? :D Maybe "Fix bug with reusing connections after error"?

(I know there's another bugfixes above and I've regretted it lol.)

File: CONTRIBUTORS.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Contributions to the urllib3 project
+﻿# Contributions to the urllib3 project
```
Comment (shazow) [inline CONTRIBUTORS.txt]: Hm what changed here? I guess line endings? Weird.

Comment (jlatherfold) [inline CONTRIBUTORS.txt]: I dunno, but those are my details and please add them whatever works best for you.

File: CONTRIBUTORS.txt (hunk 1 above)
Comment (jlatherfold) [inline CONTRIBUTORS.txt]: Ok done. Feel free to change it if you can come up with something that fits better.

**Comment by shazow** [comment]
Excellent, welcome! :custard:

**Comment by jlatherfold** [comment]
Cheers...

File: CONTRIBUTORS.txt (hunk 2 above)
Comment (Disassem) [inline CONTRIBUTORS.txt]: This is BOM in begin of file.

**Pull request merged**